specs/bls_signature.md

```diff
@@ -59,7 +59,7 @@ We require:
 * `a_flag2 == b_flag2 == c_flag2 == 0`
 * `c_flag1 == 1`
 * if `b_flag1 == 1` then `a_flag1 == x1 == x2 == 0` and `(z1, z2)` represents the point at infinity
-* if `b_flag1 == 0` then `(z1, z2)` represents the point `(x1 * i + x2, y)` where `y` is the valid coordinate such that the imaginary part `y_im` of `y` satisfies `(y_im * 2) // q == a_flag1`
+* if `b_flag1 == 0` then `(z1, z2)` represents the point `(x1 * i + x2, y)` where `y` is the valid coordinate such that the imaginary part `y_im` of `y` satisfies `(y_im * 2) // q == a_flag1`. If `y_im` happens to be `0`, use the real part to set `(y_re * 2) // q == a_flag1`.
```

Comment:
Not sure how to find an instance of (x, y) whose y_im is zero for the test case 😓.

Reply:
Now that I think about it, it may actually be impossible to find a test case..... so it's more a "just in case" rule.

I'm inclined to say don't worry about it much because we'll walk through all of this again once the BLS standardization process gets further.